Mathlib/CategoryTheory/Functor/KanExtension/Adjunction.lean

```diff
@@ -53,9 +53,30 @@ instance (F : C ⥤ H) : (L.lan.obj F).IsLeftKanExtension (L.lanUnit.app F) := b
 then `L.lan.obj G` is a pointwise left Kan extension of `F`. -/
 noncomputable def isPointwiseLeftKanExtensionLanUnit
     (F : C ⥤ H) [HasPointwiseLeftKanExtension L F] :
-    (LeftExtension.mk _ (L.lanUnit.app F)).IsPointwiseLeftKanExtension :=
+    (LeftExtension.mk _ (leftKanExtensionUnit L F)).IsPointwiseLeftKanExtension :=
```

Comment:
Hmm, I changed this in order to not have to create local instances of `Limits.HasColimit (CostructuredArrow.proj L X ⋙ (𝟭 (C ⥤ H)).obj F)`  in the new lemmas, but it causes problems further down in `Functor.Flat`. Should I revert it and have the loacl instances? @joelriou

Reply:
Ah, nevermind, I noticed I just need to add a `(F := F)`